### PR TITLE
Update du models description

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**
- Update description
- Rename id cards to match its name

### CODE REVIEW

File: metadata/IDCards__1__metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (irinel-birsan-uipath) [inline metadata/IDCards__1__metadata.json:1]: @vivekjhauipath do you see any problem if I rename the file from `IdCards` to `IDCards` now that the tar files are already generated?

Comment (vivekjhauipath) [reply metadata/IDCards__1__metadata.json:1]: Do, you plan to change modelName also?

Comment (vivekjhauipath) [reply metadata/IDCards__1__metadata.json:1]: @irinel-birsan-uipath : Please get this tested, I hope 2 packages don't get created, if IdCards was already synced.

**Review by vivekjhauipath — APPROVED** [review]
LGTM.
Though this will reflect only for new online onprem customers

File: metadata/IDCards__1__metadata.json (hunk 1 above)
Comment (irinel-birsan-uipath) [reply metadata/IDCards__1__metadata.json:1]: @vivekjhauipath In the file, the model name will remain `IDCards`. Only the file name was a bit different.
This is a new model which never reached the `main` branch.

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Pull request merged**